hooks/build

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+# this script is used to build a Radicale Docker image locally
+# as well as on the Docker Hub.
+
+echo "------ HOOK START - BUILD -------"
+
+BASE_IMAGE=$(head -n1 Dockerfile | cut -d" " -f2)
+docker pull $BASE_IMAGE
+BUILD_DATE="$(date --rfc-3339 seconds)"
+VERSION=$(grep VERSION ../setup.py | cut -f3 -d" " | tr -d '"')
```

Comment:
`git describe --tags` might be better here, as it will also report an offset to the latest released version. So the patched version won't map to the same version number as the vanilla version.